master/pkg/model/experiment.go

```diff
@@ -857,15 +857,17 @@ func ExitedReasonFromProto(r trialv1.TrialEarlyExit_ExitedReason) ExitedReason {
 	}
 }
 
-// ToProto converts an ExitedReason to its protobuf representation.
-func (r *ExitedReason) ToProto() experimentv1.TrialExitedEarly_ExitedReason {
+// ToSearcherProto converts an ExitedReason to its protobuf representation for searcher purposes.
+func (r *ExitedReason) ToSearcherProto() experimentv1.TrialExitedEarly_ExitedReason {
 	switch *r {
 	case Errored:
 		return *experimentv1.TrialExitedEarly_EXITED_REASON_UNSPECIFIED.Enum()
```

Comment:
similar comment here, just return `experimentv1.TrialExitedEarly_EXITED_REASON_UNSPECIFIED`, looks like `.Enum()` is just fancy syntax to get a pointer